### ISSUE TITLE
ImGUI IO assertion failure

### ISSUE THREAD

**Issue by prybicki** [mosra/magnum-integration#91]
With the most recent versions of ImGUI and Magnum, ImGUI example crashes with the following message:
```
external/imgui/imgui.cpp:7707: void ImGui::ErrorCheckNewFrameSanityChecks(): Assertion `g.IO.KeyMap[n] >= -1 && g.IO.KeyMap[n] < ((int)(sizeof(g.IO.KeysDown) / sizeof(*(g.IO.KeysDown)))) && "io.KeyMap[] contains an out of bound value (need to be 0..511, or -1 for unmapped key)"' failed.
```
Perhaps it is related to this discussion?
https://github.com/ocornut/imgui/pull/2269#issuecomment-453485633

**Comment by pezcode** [comment]
Should be fixed by https://github.com/mosra/magnum-integration/pull/89

**Comment by prybicki** [comment]
Works for me, thank you very much!

**Comment by mosra** [comment]
This got properly fixed with #93, merged as c60149c818bbdf61d97f30706f2018d968930718.
